### DISCUSSION TITLE
Displaying Battery Level of Bluetooth Headphones

### DISCUSSION THREAD

**Discussion by mortyobnoxious** [File-New-Project/EarTrumpet#715] [Ideas]
I don't know if this is out of range of your project but if implemented, this would be really great.

Directly seeing battery level of Bluetooth headphones something like this:
![image](https://user-images.githubusercontent.com/42044258/95461420-e2c4a700-097e-11eb-82e4-0ad4b70d38d7.png)

**Comment by riverar** [comment]
Neat idea, thanks!

**Comment by ItsMeAubey** [comment]
+1 to this. Right now, you have to open your bluetooth settings to see battery status. Would be really nice to have this. 

Thank you!

**Comment by nicolafarina** [comment]
You can try https://www.bluetoothgoodies.com
